grammar.y

```diff
@@ -1020,7 +1014,7 @@ expr_list:
 
 expr_list_opt:
   {
-    $$ = nil
+    $$ = Exprs{}
```

Comment:
This is actually a bug that I found. The bug was that when a function without arguments was used, it's string representation was coming with a `*` as an argument. For example, if `funcXXX()` was used, its string representation was `funcXXX(*)`, and not `funcXXX()`. 

Explanation:

The `expr_list_opt` is used in `function_call_generic` rule for the list of arguments. We could say that there are two kinds of arguments in a function: the "regular" one (e.g. `func(a,b,c)`, `func()`, ...) and the "star" argument (e.g. `func(*)`). The "regular" is represented as a list of `expr`, that is, `Exprs`. And the "star" is represented as `nil`. When a function had no arguments (e.g. `func()`), the `expr_list_opt` rule was returning `nil` which implied that the [`String`](https://github.com/tablelandnetwork/go-sqlparser/blob/main/ast.go#L1247-L1252) method was returning `func(*)`. 

Also, see [here](https://github.com/tablelandnetwork/go-sqlparser/pull/43/files#diff-a6ec253416e8e9e6846f73f5c7677d6ff6fa85f5c37317f84e608fc50ff8238eR989) that a `func(*)` will always have `Args` as a `nil` value. 

Let me know if it makes sense. It can be a bit confusing.

We never found this bug before because there's only one function with empty arguments: `pi()`.

Reply:
Makes sense.